sdks/uniswapx-sdk/src/utils/OrderQuoter.ts

```diff
@@ -275,12 +291,23 @@ export class UniswapXOrderQuoter
       return [order.order.serialize(), order.signature];
     });
 
+    // TODO: does not work with multiple different block overrides
+    const blockOverrides = orders.reduce((acc: BlockOverrides, order) => {
```

Comment:
the reduce here will just keep overwriting acc right? Maybe more clear way to describe this is with `Array.find(!!blockOverrides.number)`

Reply:
Or split into several multicalls, one for each block override